src/main/java/org/opensearch/neuralsearch/query/HybridQueryScorer.java

```diff
@@ -39,7 +39,7 @@ public final class HybridQueryScorer extends Scorer {
 
     private final Map<Query, Integer> queryToIndex;
 
-    HybridQueryScorer(Weight weight, List<Scorer> subScorers) throws IOException {
+    public HybridQueryScorer(Weight weight, List<Scorer> subScorers) throws IOException {
```

Comment:
Increase visibility of constructor for tests, previous package default was not required